packages/mux-uploader/src/layouts/block.ts

```diff
@@ -3,26 +3,47 @@ import { document } from '../polyfills';
 import '../mux-uploader-file-select';
 import { fileSelectFragment } from '../mux-uploader-file-select';
 
-export default function blockLayout(nodrop: boolean) {
-  const wrapper = nodrop ? 'div' : 'mux-uploader-drop overlay';
+function shouldRenderElement(selector: string): boolean {
+  return !document.querySelector(selector);
```

Comment:
If the elements are inside a Web Component, this won't find them and assume that the elements need to get added.
If this can get a reference to the mux-uploader element, we can at least check to see if the same document context is that contains it via `muxUploader.getRootNode()` instead of `document`.
Seems like potentially could pass `this` here https://github.com/muxinc/elements/blob/4e45cc07b03d29ed03fe09b94e28591f8ff9b886/packages/mux-uploader/src/mux-uploader.ts#L158